src/applications/claims-status/selectors/index.js

```diff
@@ -13,5 +13,9 @@ export const showClaimLettersFeature = state =>
 export const cstUseLighthouse = state =>
   toggleValues(state)[FEATURE_FLAG_NAMES.cstUseLighthouse];
 
+// 'cst_include_ddl_boa_letters'
+export const cstIncludeDdlBoaLetters = state =>
+  toggleValues(state)[FEATURE_FLAG_NAMES.cstIncludeDdlBoaLetters];
```

Comment:
Why do we need this change and the change in `featureFlagNames.js`?

Reply:
@kjduensing We're going to use it to toggle a link from the Appeals listing page soon... cc: @jerekshoe

Reply:
Ok thanks. You're saying it's not relevant to this PR, but we'll need it in the future?

Reply:
Yeah, basically. I can remove it if it seems like a bad idea to push it with this PR @kjduensing

Reply:
Cool beans, thanks for clarifying. I dunno that it's a bad idea, it just muddies the purpose of this PR a little. Up to you, mostly just trying to understand.